.github/workflows/ci.yml

```diff
@@ -31,20 +31,21 @@ jobs:
     timeout-minutes: 10
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Install Node
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version: 18
           cache: yarn
       - name: Install Dependencies
         run: yarn install --frozen-lockfile
       - name: Run Tests
-        run: yarn node-test-with-coverage
-      - uses: codecov/codecov-action@v4
-        with:
-          fail_ci_if_error: true
-          token: ${{ secrets.CODECOV_TOKEN }}
+        run: yarn node-test
+      #   run: yarn node-test-with-coverage
```

Comment:
I think enable this but put `if: github.event_name == 'push'`

And a comment with this link https://github.com/codecov/codecov-action/issues/1580 so we know to re-enable

Reply:
It looks like codecov-action@v5 should now work per that linked issue

Reply:
Indeed, but I forgot to push my last commit :smile: Should be okay now.